app/src/main/java/com/dessalines/rankmyfavs/db/FavList.kt

```diff
@@ -32,6 +32,10 @@ data class FavList(
         name = "description",
     )
     val description: String? = null,
+    @ColumnInfo(
+        name = "tier_list_initialized",
+    )
+    val tierListInitialized: Boolean = false,
```

Comment:
I greatly doubt this column is necessary. In the code you can check if the row in your new table exists firsts, or have all the code do UPSERTs to your new table.

Reply:
The reason I've done it like this is because I want it to be a valid scenario for the user to remove all the tier lists, and in that case a query with the favlist id would return the same result in both cases and initialize the tiers again, when in that case I don't want them to be initialized. The same with Upserts I think.